api/src/main/resources/application.yaml

```diff
@@ -154,10 +154,14 @@ resilience4j.retry:
 #Endpoint properties
 endpoint:
   grad-trax-api:
+    url: ${GRAD_TRAX_API}api/v1/trax/school/common
```

Comment:
please move 'url: ${GRAD_TRAX_API}api/v1/trax/school/common' under its own subheading similar to commonschool-by-mincode. For example:

```
grad-trax-api:
  all-commonschools:
    url: ${GRAD_TRAX_API}api/v1/trax/school/common 
```